controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/DeploymentIssueReporter.java

```diff
@@ -8,52 +8,35 @@
 import com.yahoo.vespa.hosted.controller.api.application.v4.model.TenantType;
 import com.yahoo.vespa.hosted.controller.api.identifiers.PropertyId;
 import com.yahoo.vespa.hosted.controller.api.identifiers.TenantId;
-import com.yahoo.vespa.hosted.controller.api.integration.Contacts;
-import com.yahoo.vespa.hosted.controller.api.integration.Contacts.UserContact;
-import com.yahoo.vespa.hosted.controller.api.integration.Issues;
-import com.yahoo.vespa.hosted.controller.api.integration.Issues.Classification;
-import com.yahoo.vespa.hosted.controller.api.integration.Issues.Issue;
-import com.yahoo.vespa.hosted.controller.api.integration.Issues.IssueInfo;
-import com.yahoo.vespa.hosted.controller.api.integration.Properties;
+import com.yahoo.vespa.hosted.controller.api.integration.organization.DeploymentIssues;
+import com.yahoo.vespa.hosted.controller.api.integration.organization.IssueId;
+import com.yahoo.vespa.hosted.controller.api.integration.organization.User;
 import com.yahoo.vespa.hosted.controller.application.DeploymentJobs;
 
 import java.time.Duration;
-import java.time.Instant;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 import java.util.NoSuchElementException;
 import java.util.Optional;
-import java.util.regex.Pattern;
-import java.util.stream.Collectors;
-
-import static com.yahoo.vespa.hosted.controller.api.integration.Contacts.Category.admin;
-import static com.yahoo.vespa.hosted.controller.api.integration.Issues.IssueInfo.Status.done;
+import java.util.logging.Level;
 
 /**
- * Maintenance job which creates Jira issues for tenants when they have jobs which fails continuously 
- * and escalates issues which are not handled.
+ * Maintenance job which files issues for tenants when they have jobs which fails continuously
+ * and escalates issues which are not handled in a timely manner.
```

Comment:
Quite the mouthful. Consider rephrasing :-)

Reply:
Will do.

Reply:
In a separate PR.